src/Files/UserControls/SidebarControl.xaml

```diff
@@ -22,14 +23,16 @@
     mc:Ignorable="d">
     <muxc:NavigationView.Resources>
         <ResourceDictionary>
+            <converters:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" />
             <MenuFlyout x:Key="SidebarContextMenu">
                 <ToggleMenuFlyoutItem IsChecked="{Binding ShowFavoritesSection, Mode=TwoWay}" Text="{helpers:ResourceString Name=SidebarFavorites}" />
                 <ToggleMenuFlyoutItem IsChecked="{Binding ShowLibrarySection, Mode=TwoWay}" Text="{helpers:ResourceString Name=SidebarLibraries}" />
                 <ToggleMenuFlyoutItem IsChecked="{Binding ShowDrivesSection, Mode=TwoWay}" Text="{helpers:ResourceString Name=Drives}" />
                 <ToggleMenuFlyoutItem IsChecked="{Binding ShowCloudDrivesSection, Mode=TwoWay}" Text="{helpers:ResourceString Name=SidebarCloudDrives}" />
                 <ToggleMenuFlyoutItem IsChecked="{Binding ShowNetworkDrivesSection, Mode=TwoWay}" Text="{helpers:ResourceString Name=SidebarNetworkDrives}" />
                 <ToggleMenuFlyoutItem IsChecked="{Binding ShowWslSection, Mode=TwoWay}" Text="{helpers:ResourceString Name=WSL}" />
-                <ToggleMenuFlyoutItem IsChecked="{Binding ShowFileTagsSection, Mode=TwoWay}" Text="{helpers:ResourceString Name=FileTags}" />
+                <ToggleMenuFlyoutItem IsChecked="{Binding ShowFileTagsSection, Mode=TwoWay}" Text="{helpers:ResourceString Name=FileTags}"
+                  Visibility="{Binding AreFileTagsEnabled, Mode=OneWay, Converter={StaticResource BoolToVisibilityConverter}}"                    />
```

Comment:
Is it possible to use x:Load?

Reply:
> Is it possible to use x:Load?

Currently not, because the context menu is in the resource dictionary and x:Load does not work there.